docker-compose.env.yml

```diff
@@ -202,39 +202,5 @@ services:
         a3SNje4UXN+/1xUfQskxafP8CKVSr8xxtwzSureiskb5/98moAiutpUtp15yyAm0
         rwIDAQAB
         -----END PUBLIC KEY-----
-  matomo:
```

Comment:
Git won't let me put a comment where I want to. On line 81 there is a reference to matomo in a Drupal URL that should probably go away.

Reply:
Good catch.  I did cleanup of that, and the `.env` file, since they were easy to do, and pushed changes to this PR